README.md

```diff
@@ -20,373 +41,6 @@ This repository has several major subfolders:
-
-```sh
-brew install pgcli
-```
-
-Thanks to variables set in the `.envrc`, connecting is simple:
-
-```console
-$ pgcli
-Server: PostgreSQL 11.6 (Debian 11.6-1.pgdg90+1)
-Version: 2.2.0
-Chat: https://gitter.im/dbcli/pgcli
-Home: http://pgcli.com
-postgres@localhost:postgres> SHOW server_version;
-+-------------------------------+
-| server_version                |
-|-------------------------------|
-| 11.6 (Debian 11.6-1.pgdg90+1) |
-+-------------------------------+
-SHOW
-Time: 0.016s
-postgres@localhost:postgres>
-```
-
-## Testing
-
-Run all tests other than Cypress in the project using `scripts/dev test`.
-
-### Server tests
-
-- Run `scripts/dev test:go` to run all local-only server-side tests. This requires the database to be running first. Use `scripts/dev up:backend` to start it.
-- Run `scripts/dev test:go:only [path to package folder]` (e.g. `scripts/dev test:go:only "./pkg/cedar/intake`) to run server-side tests for a specific folder. Depending on the tests being run, this may require the database to be running, as above.
-- Run `scripts/dev test:go:long` to run all server-side tests, including ones that contact external services.
-
-### JS Tests
-
-Run `scripts/dev test:js`.
-
-### Cypress tests (End-to-end integration tests)
-
-There are multiple ways to run the Cypress tests:
-
-- Run `yarn cypress run` to run the tests in the CLI.
-- To have a more interactive experience, you can instead run `yarn cypress open`.
-  - Windows+WSL users will need some additional setup to run graphical applications from within WSL.
-    - Option 1: Use the preview features available in Windows Insiders build. See [Microsoft docs](https://docs.microsoft.com/en-us/windows/wsl/tutorials/gui-apps).
-    - Option 2: Set up an X server on Windows and configure WSL to use it. See [this article](https://wilcovanes.ch/articles/setting-up-the-cypress-gui-in-wsl2-ubuntu-for-windows-10/) for details.
-  - Note: the database, frontend, and backend must be running prior to starting the Cypress tests. Use `scripts/dev up` to start them.
-  - Before each testing run, run `scripts/dev db:clean && scripts/dev db:seed` to reset the database to a pre-seeded state.
-  - The `APP_ENV` environment variable should be set to `test` in `.envrc.local`. After creating `.envrc.local` if necessary and adding `APP_ENV=test` to it, run `direnv allow` to enable it. (See [instructions above](#direnv) on `direnv` usage)
-  - Running `login.spec.js` requires the environment variables `OKTA_TEST_USERNAME`, `OKTA_TEST_PASSWORD`, and `OKTA_TEST_SECRET` to be set in `.envrc.local`. The values can be found in 1Password, as mentioned in the [Authentication section](#authentication).
-- `APP_ENV=test ./scripts/run-cypress-test-docker` : Run the Cypress tests,
-  database, migrations, backend, and frontend locally in Docker, similar to how
-  they run in CI. Running the tests in this way takes time, but is useful
-  for troubleshooting integration test failures in CI.
-
 ## Optional Setup
```

Comment:
Do you think it's even worth breaking out Optional Setup into its own doc? I really like the direction you went with these changes (that the main README just contains high-level project info, what software we're using, how the code is laid out, etc.)

I could honestly keep it here just as well, but wanted to ask!

Reply:
It feels like they should be separated to keep the main README clean, I'm just not sure whether each subsection of Optional Setup needs its own doc, or whether to split them out into a bunch of really small docs, or what.

Reply:
Going to keep them in the main README for now so we can get this PR done with. Might revisit this later.